src/main/java/org/spongepowered/common/config/SpongeConfig.java

```diff
@@ -457,6 +459,8 @@ public void setDumpThreadsOnWarn(boolean dumpThreadsOnWarn) {
         @Setting(value = GENERAL_CHUNK_LOAD_OVERRIDE,
                 comment = "Forces Chunk Loading on provide requests (speedup for mods that don't check if a chunk is loaded)")
         private boolean chunkLoadOverride = false;
+        @Setting(value = GENERAL_LOCALE, comment = "IEFT localization string (e.g. \"en_US\")")
+        private String locale = "en-US";
```

Comment:
Example has an underscore, this doesn't - most people working with Minecraft translations are used to underscores, from what I've seen.